project/sites/boundarycommission/themes/boundarycommission_theme/src/scss/components.scss

```diff
@@ -54,3 +54,109 @@
     }
   }
 }
+
+.homepage--pull-quote {
+  background: resolve('front-banner.png') no-repeat 50%;
+  height: 30rem;
+
+  .container-wrap {
+    padding: 9.4rem 0;
+  }
+
+  blockquote {
+    @include adjust-font-size-to(30px);
+    font-weight: bold;
+    max-width: 45rem;
+
+    p {
+      color: $white;
+    }
+
+    &::before,
+    &::after {
+      background: none;
+      padding: 0;
+    }
+  }
+}
+
+.region-front-welcome {
+  border-bottom: solid $brand-primary 2px;
+
+  p {
+    font-size: 2.1rem;
+    text-align: left;
+    display: flex;
+    line-height: 3rem;
+    width: 100%;
+
+    .btn {
+      width: 26rem;
+    }
+  }
+}
+
+#top-area {
+  display: flex;
+  justify-content: space-evenly;
+  margin-bottom: 2rem;
+
+  .column {
+    max-width: 28rem;
+    border: solid 2px $brand-primary;
+
+    .content {
+      padding: 0;
+    }
+  }
+
+  .media-image {
+    margin: 0;
+  }
+}
+
+#bottom-area {
+  @include breakpoint($tab) {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 4rem;
+    margin-top: 4rem;
+  }
+
+  .column {
+    width: auto;
+    float: none;
+
+    @include breakpoint($tab) {
+      flex-basis: 46%;
+    }
+  }
+}
+
+#footer {
+  .container-wrap {
+    display: block;
+  }
+
+  @include breakpoint($tab) {
+    padding: 2%;
+  }
+
+  .footer-menu li {
```

Comment:
This code should come from Unity theme now https://github.com/dof-dss/nicsdru_unity_theme/blob/development/src/scss/2_layout/_footer.scss#L36 so you shouldn't need to re-add it here.

Reply:
Done, forgot about this! Thanks Adam